source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -0,0 +1,250 @@
+===================================
+Document Data Format: Extended JSON
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+JSON is a data format that encodes the values of objects, arrays, numbers, and 
+so on. The **Extended JSON** format adds special syntax to JSON objects to 
+represent field type information that directly corresponds to each type in 
+BSON, the format that MongoDB uses to store data.
+
+This guide explains the following topics:
+
+- the different MongoDB Extended JSON formats
+- how to use the BSON library to convert between Extended JSON and Java objects
+
+For more information on the difference between these formats, see our
+`article on JSON and BSON <https://www.mongodb.com/json-and-bson>`__.
+
+Extended JSON Formats
+---------------------
+
+MongoDB Extended JSON represents BSON in different formats to handle
+specific use cases. These formats differ in the level of prioritization in
```

Comment:
Can you try to re-word the sentence starting with "These formats".  I think I know what you mean but I think casual readers will not understand what you mean

Reply:
Rewrote the paragraph f154afda39c6d5d83d850a77618fb0a8673b4eb5